src/lib/logger.config.ts

```diff
@@ -7,4 +7,6 @@ export class LoggerConfig {
   disableConsoleLogging?: boolean;
   httpResponseType?: 'arraybuffer' | 'blob' | 'text' | 'json';
   enableSourceMaps?: boolean;
+  /** Custom timestamp formatter. Defaults to toISOString if undefined */
+  timestampFormatter?: (Date) => string;
```

Comment:
We should not allow a config property to be a method. If we are going to use the config, we should support a set of formatting types. much like the predefined formats supported by the angular date pipe (https://angular.io/api/common/DatePipe#pre-defined-format-options).

Reply:
ah ok, didn't know. What is the rationale behind not using a function? Security?